The error happened again once in a 24h pressure test run and was successfully fixed by this PR 🎉
```
2026-01-15 15:44:12 - lab-data - WARNING: Timeout to return plate to location 1:1
Unexpected response while checking status: Expected response prefix 'be', got response 'ba 83'
Traceback (most recent call last):
  File "C:\Users\fungtor\repos\lab-data\.venv\lib\site-packages\cytomat\serial_port.py", line 164, in issue_status_command
    response = self.check_prefix_and_strip(self.communicate(command), expected_prefix=command[3:5])
  File "C:\Users\fungtor\repos\lab-data\.venv\lib\site-packages\cytomat\serial_port.py", line 104, in check_prefix_and_strip
    raise UnexpectedResponse(f"Expected response prefix '{expected_prefix}', got response '{response}'")
cytomat.errors.UnexpectedResponse: Expected response prefix 'be', got response 'ba 83'
Trying again
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "open-cytomat"
-version = "0.3.2"
+version = "0.3.3"
 description = ""
 authors = [
     "Niklas Mertsch <niklas.mertsch@stud.uni-goettingen.de>",

--- a/src/cytomat/serial_port.py
+++ b/src/cytomat/serial_port.py
@@ -1,5 +1,4 @@
 import re
-import traceback
 from threading import Lock
 
 from serial import Serial
@@ -163,9 +162,7 @@ class SerialPort:
         try:
             response = self.check_prefix_and_strip(self.communicate(command), expected_prefix=command[3:5])
         except UnexpectedResponse as e:
-            print(f"Unexpected response while checking status: {e}")
-            traceback.print_exc()
-            print("Trying again")
+            print(f"Unexpected response while checking status: {e}. Trying again.")
             response = self.check_prefix_and_strip(self.communicate(command), expected_prefix=command[3:5])
 
         return response

